packages/views/screens/managers/wallet/explorerLayout.js

```diff
@@ -0,0 +1,68 @@
+/* istanbul ignore file */
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { isEmpty } from '@common/utilities/helpers';
+import { selectActiveToken, selectSettings } from '@common/store/selectors';
+import { selectSearchParamValue } from '@screens/router/searchParams';
+import TabsContainer from '@basics/tabs/tabsContainer/tabsContainer';
+import Overview from './overview';
+import DelegateTab from './delegateProfile';
+import VotesTab from './votes';
+import Transactions from './transactions';
+
+const ExplorerLayout = ({
+  t, account, history,
+}) => {
+  const activeToken = useSelector(selectActiveToken);
+  const { discreetMode } = useSelector(selectSettings);
+  const address = selectSearchParamValue(history.location.search, 'address');
+
+  useEffect(() => {
+    account.loadData();
+  }, [address]);
+
+  if (!account || !account.data || isEmpty(account.data)) {
+    return (<div />);
+  }
```

Comment:
```suggestion
  const isAccountEmpty = useMemo(() => isEmpty(account.data), [account.data])
  if (!account?.data || isAccountEmpty) {
    return (<div />);
  }
```

I think this would reduce the number of boolean checks and also, `useMemo` would help memorize the result of `isEmpty` until there is a change to `account.data`